docs/edge-authorizers/runtime-flow.mdx

```diff
@@ -20,61 +20,66 @@ The runtime configuration is automatically retrieved from the control plane and
 
 ```yaml
 ---
-logging:
-  prod: true
-  log_level: info
-
-directory_service:
-  path: /app/db/directory.db
-
-api:
-  grpc:
-    connection_timeout_seconds: 2
-    listen_address: '0.0.0.0:8282'
-    certs:
-      tls_key_path: '/app/certs/grpc.key'
-      tls_cert_path: '/app/certs/grpc.crt'
-      tls_ca_cert_path: '/app/certs/grpc-ca.crt'
-  gateway:
-    listen_address: '0.0.0.0:8383'
-    allowed_origins:
-      - https://*.mydomain.com
-    certs:
-      tls_key_path: '/app/certs/gateway.key'
-      tls_cert_path: '/app/certs/gateway.crt'
-      tls_ca_cert_path: '/app/certs/gateway-ca.crt'
-  health:
-    listen_address: '0.0.0.0:8484'
-
-opa:
-  discovery:
-    bundles:
-      '<POLICY_ID>':
-        persist: false
-        polling:
-          max_delay_seconds: 120
-          min_delay_seconds: 60
-        resource: '<POLICY_ID>/bundle.tar.gz'
-        service: aserto-bundler
-        signing:
-        size_limit_bytes: 0
-    plugins:
-      aserto_edge:
-        addr: authorizer.prod.aserto.com:8443
-        apikey: '<REDACTED directory API key>'
-        enabled: false
-        insecure: false
-        page_size: 100
-        sync_interval: 60
-        timeout: 10
-    services:
-      aserto-bundler:
-        credentials:
-          bearer:
-            token: '<REDACTED> registry download API key'
-        response_header_timeout_seconds: 60
-        url: https://bundler.prod.aserto.com/<TENANT_ID>/
+tenant_id: <TENANT-ID>
+policy_name: <POLICY-NAME>
+instance_label: <INSTANCE-LABEL>
 
+authorizer:
+  logging:
+    prod: true
+    log_level: info
+  
+  directory_service:
+    path: /app/db/directory.db
+  
+  api:
+    grpc:
+      connection_timeout_seconds: 2
+      listen_address: '0.0.0.0:8282'
+      certs:
+        tls_key_path: '/app/certs/grpc.key'
+        tls_cert_path: '/app/certs/grpc.crt'
+        tls_ca_cert_path: '/app/certs/grpc-ca.crt'
+    gateway:
+      listen_address: '0.0.0.0:8383'
+      allowed_origins:
+        - https://*.mydomain.com
+      certs:
+        tls_key_path: '/app/certs/gateway.key'
+        tls_cert_path: '/app/certs/gateway.crt'
+        tls_ca_cert_path: '/app/certs/gateway-ca.crt'
+    health:
+      listen_address: '0.0.0.0:8484'
+  
+  opa:
+    discovery:
+      bundles:
+        '<POLICY_ID>':
+          persist: false
+          polling:
+            max_delay_seconds: 120
+            min_delay_seconds: 60
+          resource: '<POLICY_ID>/bundle.tar.gz'
```

Comment:
seems like we use a different path here?  <POLICY_NAME>/<INSTANCE_LABEL>/opa or something like that?